rapidjson.cpp

```diff
@@ -434,28 +434,26 @@ RawJSON_dealloc(RawJSON* self)
 }
 
 
-static int
-RawJSON_init(RawJSON* self, PyObject* args, PyObject* kwds)
+static PyObject *
+RawJSON_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
```

Comment:
Please follow overall stylistic convention, using `PyObject* foo` instead of `PyObject *foo`: although it's not my preferred style, but I tried to adhere to the one originally used by Ken